frontend/redux/nodes/entities/hosts/helpers.js

```diff
@@ -4,14 +4,20 @@ export const parseEntityFunc = (host) => {
   const { network_interfaces: networkInterfaces } = host;
   const networkInterface = networkInterfaces && find(networkInterfaces, { id: host.primary_ip_id });
 
-  let clockSpeed = null;
-  let clockSpeedFlt = null;
   let hostCpuOutput = null;
-
-  if (host && host.cpu_brand) {
-    clockSpeed = host.cpu_brand.split('@ ')[1] || host.cpu_brand.split('@')[1];
-    clockSpeedFlt = parseFloat(clockSpeed.split('GHz')[0].trim());
-    hostCpuOutput = `${host.cpu_physical_cores} x ${Math.floor(clockSpeedFlt * 10) / 10} GHz`;
+  if (host) {
```

Comment:
Would it be possible to just do away with the parsed variant, and to print the string as is?

If not, the current implementation LGTM

Reply:
Not with the current design
<img width="338" alt="screen shot 2017-11-13 at 7 35 44 pm" src="https://user-images.githubusercontent.com/575602/32761736-e01dc8ce-c8a9-11e7-8aed-33b4db911829.png">